src/domains/checkout-app/05_checkout_function.tf

```diff
@@ -95,129 +95,3 @@ module "checkout_function" {
 
   tags = var.tags
 }
-
-resource "azurerm_monitor_autoscale_setting" "checkout_function" {
-  count = var.checkout_enabled && var.env_short != "d" ? 1 : 0
-
-  name                = format("%s-autoscale", module.checkout_function[0].name)
-  resource_group_name = azurerm_resource_group.checkout_be_rg[0].name
-  location            = var.location
-  target_resource_id  = module.checkout_function[0].app_service_plan_id
-
-  profile {
-    name = "default"
-
-    capacity {
-      default = var.checkout_function_autoscale_default
-      minimum = var.checkout_function_autoscale_minimum
-      maximum = var.checkout_function_autoscale_maximum
-    }
-
-    rule {
-      metric_trigger {
-        metric_name              = "Requests"
-        metric_resource_id       = module.checkout_function[0].id
-        metric_namespace         = "microsoft.web/sites"
-        time_grain               = "PT1M"
-        statistic                = "Average"
-        time_window              = "PT5M"
-        time_aggregation         = "Average"
-        operator                 = "GreaterThan"
-        threshold                = 4000
-        divide_by_instance_count = false
-      }
-
-      scale_action {
-        direction = "Increase"
-        type      = "ChangeCount"
-        value     = "2"
-        cooldown  = "PT5M"
-      }
-    }
-
-    rule {
-      metric_trigger {
-        metric_name              = "Requests"
-        metric_resource_id       = module.checkout_function[0].id
-        metric_namespace         = "microsoft.web/sites"
-        time_grain               = "PT1M"
-        statistic                = "Average"
-        time_window              = "PT5M"
-        time_aggregation         = "Average"
-        operator                 = "LessThan"
-        threshold                = 3000
-        divide_by_instance_count = false
-      }
-
-      scale_action {
-        direction = "Decrease"
-        type      = "ChangeCount"
-        value     = "1"
-        cooldown  = "PT20M"
-      }
-    }
-  }
-}
-
-# Availability: Checkout functions & pagopa-proxy
-resource "azurerm_monitor_scheduled_query_rules_alert" "checkout_availability" {
```

Comment:
pay attention, this alert is related to pagopa-proxy and not checkout function

Reply:
restored